src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs

```diff
@@ -11,7 +11,7 @@
    ))
 
 
-(def ^:private initial-rows-per-page 10)
+(def ^:private default-initial-rows-per-page 10)
```

Comment:
I think we should make 20 the default. I think this makes sense for most of our tables (everything except the workspaces list).